_pytest/__init__.py

```diff
@@ -1,2 +1,4 @@
-#
-__version__ = '2.8.0.dev5'
+# coding: utf-8
+# file generated by setuptools_scm
+# don't change, don't track in version control
+version = '2.7.4.dev394+ng2575053.d20150916'
```

Comment:
After I execute `python setup.py develop` this file is getting changed in my local copy. @RonnyPfannschmidt, could you comment on this please? :smile: